packages/fast-usdc-contract/src/exos/settler.ts

```diff
@@ -57,6 +57,12 @@ import { asMultiset } from '../utils/store.ts';
 import type { LiquidityPoolKit } from './liquidity-pool.js';
 import type { StatusManager } from './status-manager.js';
 
+const FORWARD_TIMEOUT = {
+  sec: 10n * 60n,
+  p: '10m',
+} as const;
+harden(FORWARD_TIMEOUT);
```

Comment:
interesting pattern! It could be better to have a single value to change, but there's a low risk that someone will change one and not the other accordingly